unittest/run_tests.sh

```diff
@@ -10,9 +10,10 @@ cd $TEST_DIR/../..
 passed_tests=()
 failed_tests=()
 
-for test in `cd $TEST_DIR; find . -name \*_test.py -print`; do
+for test in `cd $TEST_DIR; find buildtool -name \*_test.py -print`; do
   echo "Running $test"
-  PYTHONPATH=$TEST_DIR/../pylib:$TEST_DIR/../dev python $TEST_DIR/$test
+#  PYTHONPATH=$TEST_DIR/../pylib:$TEST_DIR/../dev python $TEST_DIR/$test
```

Comment:
Is this here for documentation purposes?

Reply:
oops